src/server/routes/api/index.js

```diff
@@ -109,3 +112,36 @@ export function handleSqsDataProcessing(sqsClient, type) {
     });
   };
 }
+
+export function getRefileErrors(req, res, next) {
+  // Read local .env file. The environment variables will be assigned with process.env in the beginning
+  dotEnv.config();
+
+  const client = new NyplApiClient({
+    base_url: 'https://dev-platform.nypl.org/api/v0.1/',
+    oauth_key: config.oauth.refileRequestId,
+    oauth_secret: process.env.REFILE_REQUEST_SECRET,
```

Comment:
Same question really. But, again, I don't mind, just wanted to get a better sense of your thoughts on it.